pkg/auth/ldaputil/testclient/testclientconfig.go

```diff
@@ -0,0 +1,37 @@
+package testclient
+
+import "github.com/openshift/origin/pkg/auth/ldaputil/ldapclient"
+
+// fakeConfig regurgitates internal state in order to conform to Config
+type fakeConfig struct {
+	client       ldapclient.Client
+	bindDN       string
+	bindPassword string
+	ldapURL      string
+}
+
+// NewConfig creates a new Config impl that regurgitates the given data
+func NewConfig(client ldapclient.Client, bindDN, bindPassword, ldapURL string) ldapclient.Config {
+	return &fakeConfig{
+		client:       client,
+		bindDN:       bindDN,
+		bindPassword: bindPassword,
+		ldapURL:      ldapURL,
+	}
+}
+
+func NewConnectingConfig(client ldapclient.Client) ldapclient.Config {
```

Comment:
I didn't like having empty strings passed in all the time to `NewConfig`, is there a better way to do that?